package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "statamic-image-optimize",
+  "private": true,
+  "description": "Image optimization after upload",
+  "scripts": {
+    "development": "mix",
+    "watch": "mix watch",
+    "production": "mix --production"
+  },
+  "author": "JustBetter",
+  "dependencies": {
+    "cross-env": "^7.0.3",
```

Comment:
Perhaps separate dependencies and dev dependencies. e.g. cross-env, laravel-mix en postcss